acceptance/config/nodes/lucidx64.yaml

```diff
@@ -3,15 +3,15 @@ HOSTS:
     roles:
```

Comment:
if we're renaming these anyway, can we please name them after the os name? we shouldn't have to know OS release codenames to run tests.  
actually, since we're renaming them all, why don't we follow the vmpooler template naming conventions?
so this one would be "ubuntu-lucid-x86_64"